CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,61 @@
+# Changelog since v2.1.0
+
+## New Features
+
+### Snapshot APIs
+
+- The validation for volume snapshot objects (VolumeSnapshot and VolumeSnapshotContent) is getting more strict. Due to backwards compatibility this change will occur over multiple releases. The following key changes are highlighted.
```

Comment:
The README table and top level usage steps should be updated with the webhook.

In the compatibility table, I would actually just remove all the csi spec versions under < 1.2 since it's backwards compatible.

Reply:
Sure